### ISSUE TITLE
Iterating over test cases fails because of no sort key

### ISSUE THREAD

**Issue by sgrognard** [kiwitcms/Kiwi#194]
### Description of problem
```
for testcase in self.testcases:
  File ".../tcms_api/containers.py", line 796, in __iter__self._items, key=lambda x: self._object.sortkey(x)):
     TypeError: '<' not supported between instances of 'NoneType' and 'NoneType'
```
### Additional info
Since it looks like no sort key is set, modifying
```
        for testcase in sorted(
                self._items, key=lambda x: self._object.sortkey(x)):
            yield testcase
```
to
```
        for testcase in self._items:
            yield testcase
```
naturally "solves" the issue.

**Comment by atodorov** [comment]
@sgrognard can you post some of your code which calls the API and triggers the problem. We also need to know what data triggers this traceback.

**Comment by atodorov** [comment]
This has been fixed during the RPC refactoring released with version 4.0.0. I've just added a test for it and will close this issue once the test is merged.
